varnish-sys/src/vcl/ws.rs

```diff
@@ -78,6 +88,9 @@ impl<'a> Workspace<'a> {
 
     /// Check if a byte slice is allocated in the workspace
     pub fn is_slice_allocated(&self, slice: &[u8]) -> bool {
+        // FIXME: why check for slice size + 1?  In case of a string,
```

Comment:
Looking at it now. I'm thing we can just check whether the pointer points to an address inside the allocated area of the workspace and call it a day

Reply:
sure, that could also work